web/src/testdata.tsx

```diff
@@ -0,0 +1,136 @@
+
```

Comment:
I think this can just be a regular `.ts` file since it doesn't have any jsx in it? Doesn't super matter though.

Reply:
but it might have jsx in the future! i don't want to have to change the name of the file every time we change the contents.

Reply:
yah! fair enough :)